src/test/kotlin/it/pagopa/wallet/scheduler/services/TimeStampRedisTemplateTest.kt

```diff
@@ -3,49 +3,51 @@ package it.pagopa.wallet.services
 import it.pagopa.wallet.scheduler.services.TimestampRedisTemplate
 import java.time.Duration
 import java.time.Instant
-import org.junit.jupiter.api.Assertions
 import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.extension.ExtendWith
 import org.mockito.junit.jupiter.MockitoExtension
 import org.mockito.kotlin.*
-import org.springframework.data.redis.core.RedisTemplate
-import org.springframework.data.redis.core.ValueOperations
+import org.springframework.data.redis.core.ReactiveRedisTemplate
+import org.springframework.data.redis.core.ReactiveValueOperations
 import org.springframework.test.context.TestPropertySource
+import reactor.core.publisher.Mono
+import reactor.test.StepVerifier
 
 @ExtendWith(MockitoExtension::class)
 @TestPropertySource(locations = ["classpath:application-test.properties"])
-class TimeStampRedisTemplateTest {
-    private val redisTemplate: RedisTemplate<String, Instant> = mock()
-    private val valueOps: ValueOperations<String, Instant> = mock()
+class ReactiveTimestampRedisTemplateTest {
```

Comment:
```suggestion
class TimestampRedisTemplateTest {
```
I would keep this class name aligned with both the file name and the tested class name